std.ddoc

```diff
@@ -15,16 +15,23 @@ DDOC = <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
 <script src="/js/codemirror-compressed.js"></script>
 <script src="/js/run.js" type="text/javascript"></script>
 <script type="text/javascript">
-/* Declaration nesting level */
-var nestingLevel = 0;
 /* Symbol definition */
 var lastSymbolDefined = '';
-var values = new Array;
+var values = [];
+//
+var lastName = function(a){
+    var pos = a.lastIndexOf('.');
+    return a.slice(pos + 1);
+}
 //
 function defineSymbol(sym) {
-  if (nestingLevel > 0) return;
+  // ignore ditto overloads
   if (sym == lastSymbolDefined) return;
-  if (true) lastSymbolDefined = sym;
+  // ignore scoped symbols, but show module-level overloads
+  var pos = sym.indexOf('.');
+  if (pos >= 0 && (sym.lastIndexOf('.') != pos || isNaN(lastName(sym)))) return;
+  // Note: empty comment avoids macro def
```

Comment:
The next line is `/**/ lastSymbolDefined = sym;`. Without the `/**/`, ddoc would end the current macro definition and start a new macro called `lastSymbolDefined`. Andrei used `if (true)` instead, but I thought an empty comment was slightly better. IMO ddoc should not allow macro definitions when there is indentation.